test-resources/integration-tests/start/start-happy.test.ts

```diff
@@ -1,51 +1,54 @@
 import { stackOutputs } from "../helpers/cloudformation";
-import { signedFetch } from "../helpers/fetch";
 import { kmsClient } from "../helpers/kms";
 import { JweDecrypter } from "../helpers/jwe-decrypter";
 import { JwtVerifierFactory, ClaimNames } from "../helpers/jwt-verifier";
 import { getParametersValues } from "../../headless-core-stub/utils/src/parameter/get-parameters";
-
+import { base64Encode } from "../../headless-core-stub/utils/src/base64/index";
+import { Logger } from "@aws-lambda-powertools/logger";
+import { signedFetch } from "../helpers/fetch";
 describe("happy path core stub start endpoint", () => {
-    let authenticationAlg;
-    let publicSigningJwkBase64;
-    let testHarnessExecuteUrl;
-    let jweDecrypter;
+    let authenticationAlg: string;
+    let testHarnessExecuteUrl: string;
+    let jweDecrypter: JweDecrypter;
 
-    const jwtVerifierFactory = new JwtVerifierFactory();
+    const jwtVerifierFactory = new JwtVerifierFactory(new Logger());
     const clientId = "ipv-core-stub-aws-headless";
     const aud = "https://test-aud";
-    const iss = "https://test-issuer";
+    let iss: string;
 
     beforeAll(async () => {
         const { TestHarnessExecuteUrl, CommonStackName } = await stackOutputs(process.env.STACK_NAME);
         testHarnessExecuteUrl = TestHarnessExecuteUrl;
+        iss = TestHarnessExecuteUrl.replace(/\/+$/, "");
 
         const { CriDecryptionKey1Id: decryptionKeyId } = await stackOutputs("core-infrastructure");
 
-        ({ authenticationAlg, publicSigningJwkBase64 } = await getParametersValues([
+        ({ authenticationAlg } = await getParametersValues([
             `/${CommonStackName}/clients/${clientId}/jwtAuthentication/authenticationAlg`,
-            `/${CommonStackName}/clients/${clientId}/jwtAuthentication/publicSigningJwkBase64`,
         ]));
 
         jweDecrypter = new JweDecrypter(kmsClient, () => decryptionKeyId);
     });
 
     it("returns 200 with a valid JWT for a valid request", async () => {
-        // Decodes to {"aud":"https://test-aud","redirect_uri":"https://test-resources.review-hc.dev.account.gov.uk/callback"}
-        const defaultState =
-            "eyJhdWQiOiJodHRwczovL3Rlc3QtYXVkIiwicmVkaXJlY3RfdXJpIjoiaHR0cHM6Ly90ZXN0LXJlc291cmNlcy5yZXZpZXctaGMuZGV2LmFjY291bnQuZ292LnVrL2NhbGxiYWNrIn0="; // pragma: allowlist secret
-        const data = await signedFetch(`${testHarnessExecuteUrl}start`, {
+        const defaultState = base64Encode(
```

Comment:
Hmm I hard coded this becuase I didn't want our tests to rely on our helpers - if they break then tests for code that don't use those helpers break.

Keen to hear why you think doing it like this is better?

Reply:
When I ran this with my test-resources stack it broke, 

`redirect_uri: `https://${TEST_RESOURCES_STACK_NAME}.review-hc.dev.account.gov.uk/callback`,

Then it passed on my stack, but when I pushed it broke, so I change it back, but no longer hard-coded b'cos it not obvious before that it has to be `https://test-resources.review-hc.dev.account.gov.uk/callback`

I believe https://github.com/govuk-one-login/ipv-cri-common-lambdas/blob/main/infrastructure/lambda/template.yaml#L1779 maybe need to move into the `test-resources template` am torn on that, anyway this is why it's no longer hard-coded